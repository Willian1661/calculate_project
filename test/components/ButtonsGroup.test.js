import React from "react";
import { expect, describe, it } from '@jest/globals';
import renderer from "react-test-renderer";
import Button from "../../src/components/Button";
import ButtonsGroup from "../../src/components/ButtonsGroup";

describe("pattern div component", () => {

    it("is passing dynamic buttons", () => {
        const component = renderer.create(
            <ButtonsGroup
                groupsId={"foo"}
                htmlClass={"faa"}
            >
                <Button
                    htmlClass={"faa"}
                    data={"bar"}>{"foo"}
                </Button>
            </ButtonsGroup>
        );

        expect(component).toMatchSnapshot();
    });

});