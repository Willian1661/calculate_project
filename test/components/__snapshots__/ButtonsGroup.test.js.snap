// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pattern div component is passing dynamic buttons 1`] = `
<div
  className="faa"
  id="foo"
>
  <button
    className="button faa"
    data-value="bar"
  >
    foo
  </button>
</div>
`;
